packages/plugins/sentry/src/index.ts

```diff
@@ -80,13 +75,13 @@ export type SentryPluginOptions = {
   skip?: (args: ExecutionArgs) => boolean;
   /**
    * Indicates whether or not to skip Sentry exception reporting for a given error.
-   * By default, this plugin skips all `EnvelopError` errors and does not report it to Sentry.
+   * By default, this plugin skips all `Error` errors and does not report it to Sentry.
    */
   skipError?: (args: Error) => boolean;
 };
 
 export function defaultSkipError(error: Error): boolean {
-  return error instanceof EnvelopError;
+  return error instanceof Error;
```

Comment:
This means that it will skip any error and report nothing? 😅

Reply:
@saihaj bump

Reply:
https://github.com/n1ru4l/envelop/pull/1487/commits/049f13d277718716f48e06a5816ac43b318775b8